### PR TITLE
promote catalog persistence in collective to the current server version

### PR DESCRIPTION
**Pull request by lavenal**
This PR is to fix a bug in the collective catalog persistence file which contains a bookmark that was created by an earlier Open Liberty server but the bookmark was removed in the later Open Liberty server. To fix the problem, the catalog in the collective would be promoted to the current version if it was a default catalog created by an earlier server version.

https://github.com/OpenLiberty/open-liberty/issues/18282

### CODE REVIEW

**Comment by skasund** [comment]
L2 message review completed

**Comment by lavenal** [comment]
A combined build with test fixes: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_ixEokAQTEeyB1uNHimT5CA&tab=com.ibm.team.build.web.ui.internal.editors.result.overview.OverviewPage

File: dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops [hunk 1]
```diff
@@ -316,4 +316,8 @@ TOOLBOX_FILE_PROMOTED_TO_ENCODED_NAME.useraction=No action is required.
 
 TOOL_DATA_PROMOTED_TO_ENCODED_NAME=CWWKX1069I: Tool {0} data for user {1} was loaded from the persisted storage using its old file name and has converted to its new file name.
 TOOL_DATA_PROMOTED_TO_ENCODED_NAME.explanation=The product loaded the tool data for the specified user and tool from persisted storage using its old file name and stored a copy into the new file name. The copy in the new file name will be loaded by subsequent access to persisted storage.
-TOOL_DATA_PROMOTED_TO_ENCODED_NAME.useraction=No action is required.
\ No newline at end of file
+TOOL_DATA_PROMOTED_TO_ENCODED_NAME.useraction=No action is required.
+
+CATALOG_FILE_PROMOTED_TO_CURRENT_VERSION=CWWKX1070I: The Admin Center default catalog was loaded from COLLECTIVE persistence created by an earlier server version. The catalog has reloaded using a copy from the current server version.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops]: Change,
"The Admin Center default catalog was loaded from COLLECTIVE persistence created by an earlier server version. The catalog has reloaded using a copy from the current server version."
to,
"The Admin Center default catalog was loaded from COLLECTIVE persistence that is created by an earlier server version. The catalog is reloaded by using a copy from the current server version."

File: dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops [hunk 2]
```diff
@@ -316,4 +316,8 @@ TOOLBOX_FILE_PROMOTED_TO_ENCODED_NAME.useraction=No action is required.
...
+CATALOG_FILE_PROMOTED_TO_CURRENT_VERSION.explanation=The product converted the Admin Center default catalog to the current server version. The copy in the COLLECTIVE persistence will be loaded by subsequent access to persisted storage.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops]: Change,
"The product converted the Admin Center default catalog to the current server version. The copy in the COLLECTIVE persistence will be loaded by subsequent access to persisted storage."
to,
"The product converted the Admin Center default catalog to the current server version. The copy in the COLLECTIVE persistence is loaded by subsequent access to the persisted storage."

**Review by ManasiGandhi — CHANGES_REQUESTED** [review]
A few edit suggestions.

**Review by k8vance88 — APPROVED** [review]

File: dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops (hunk 1 above)
Comment (lavenal) [reply dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops]: Thanks and changed per review!

File: dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops (hunk 2 above)
Comment (lavenal) [reply dev/com.ibm.ws.ui/resources/com/ibm/ws/ui/internal/resources/UIMessages.nlsprops]: Thanks and changed per review comment.

**Review by ManasiGandhi — APPROVED** [review]

**Comment by lavenal** [comment]
New personal build with ID review changes: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_qPKzMAbDEeyB1uNHimT5CA&tab=com.ibm.team.build.web.ui.internal.editors.result.overview.OverviewPage

**Comment by lavenal** [comment]
#build

**Comment by lavenal** [comment]
#run-libby-bot

**Comment by lavenal** [comment]
The failure in PB is not related to the code changes.

**Pull request merged**